com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/BundleHelper.java

```diff
@@ -701,6 +701,11 @@ private boolean copyIconDPI(BobProjectProperties projectProperties, String proje
         return properties;
     }
 
+    private String derivedBundleName() {
```

Comment:
We usually put set/get in front of the names for getters/setters.
-> "getDerivedBundleName()"